src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <block class="stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </block>
+
+    <block class="stars--1">
+      <div class="stars__star-active"></div>
```

Comment:
all stars should have **only** stars__star class

```suggestion
      <div class="stars__star"></div>
```